inc/theme-hooks.php

```diff
@@ -125,3 +125,123 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+
+add_filter( 'alcatraz_primary_sidebar', 'alcatraz_sidebar_nav_menu', 5);
+/**
+ * Output the sidebar nav.
+ *
+ * @since  1.0.0
+ */
+
+function alcatraz_sidebar_nav_menu() {
+
+	global $post;
+
+	$args = array();
+
+	//defaults
+	$defaults = array(
+		'show_all'         => true,
+		'show_on_home'     => false,
+		'show_empty'       => false,
+		'sort_by'          => 'menu_order',
+		'a_heading'        => false,
+		'title'            => ''
+	);
+
+	$args = wp_parse_args( (array)$args, $defaults );
```

Comment:
No need to cast $args as an array here since you've defined it directly as an array above.